framework/src/org/checkerframework/framework/util/defaults/QualifierDefaults.java

```diff
@@ -428,7 +476,8 @@ private boolean isElementAnnotatedForThisChecker(final Element elt) {
 
         if (elementAnnotatedForThisChecker == false) {
             Element parent;
-            if (elt.getKind() == ElementKind.PACKAGE) // Must NOT look at packages.
+            if (elt.getKind() == ElementKind.PACKAGE)
```

Comment:
Even single statement "then" or "else" blocks should be in `{}` blocks. The code below is inconsistent about this.